packages/hierarchy-builder/src/test/internal/operators/grouping/AutoExpand.test.ts

```diff
@@ -0,0 +1,163 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { expect } from "chai";
+import { AutoExpand, GroupingNodeKey, HierarchyNode, InstanceHierarchyNodeProcessingParams } from "../../../../hierarchy-builder/HierarchyNode";
+import { GroupingType, ProcessedInstancesGroupingHierarchyNode } from "../../../../hierarchy-builder/internal/operators/Grouping";
+import { assignAutoExpand } from "../../../../hierarchy-builder/internal/operators/grouping/AutoExpand";
+import { createTestProcessedGroupingNode, createTestProcessedInstanceNode } from "../../../Utils";
+
+describe("AutoExpand", () => {
+  [
+    {
+      testName: "Base class grouping",
+      testParams: {
+        label: "Test label",
```

Comment:
I don't think we need `label` parameter as it's the same for all cases and we don't really care what it is. Instead, just hardcode it in each test.

Reply:
Changed.